org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -27,15 +28,27 @@
 import org.eclipse.jface.wizard.Wizard;
 import org.eclipse.ui.INewWizard;
 import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchWindow;
 import org.eclipse.ui.IWorkingSetManager;
 import org.eclipse.ui.internal.WorkbenchPlugin;
 
 public class DotnetNewWizard extends Wizard implements INewWizard {
 	private DotnetNewWizardPage wizardPage;
 	@Override
 	public void init(IWorkbench workbench, IStructuredSelection selection) {
-		// TODO
 		wizardPage = new DotnetNewWizardPage();
+
+		IWorkbenchWindow window = workbench.getActiveWorkbenchWindow();
+		if (window != null) {
```

Comment:
I don't believe you need those 2 lines as you should directly consume the selection parameter.